tensorflow/c/kernels.cc

```diff
@@ -551,3 +554,257 @@ TF_Tensor* TF_AllocateTemp(TF_OpKernelContext* context, TF_DataType dtype,
   }
   return tf_tensor;
 }
+
+tensorflow::Status EnsureSparseVariableAccess(TF_OpKernelContext* ctx,
```

Comment:
I am moving these to c_api_experimental file (ran into some build issues). I wanted to get started with PR, to get feedback.